Safe/Foldable.hs

```diff
@@ -59,15 +62,15 @@ minimumNote, maximumNote :: (Foldable t, Ord a) => String -> t a -> a
 minimumNote note = fromNote note "minimumNote on empty" . minimumMay
 maximumNote note = fromNote note "maximumNote on empty" . maximumMay
 
-minimumByMay, maximumByMay :: (Foldable t) => (a -> a -> Ordering) -> t a -> Maybe a
```

Comment:
Interesting - my preference is without the brackets, but obviously someone other than me submitting this as a patch, so they got included. I'm happy either way.